database/migrations/2016_06_29_205018_add_oauth_fields_to_users.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
@DFurnes ahhh i see. hmm i guess we can leave them there then! Just didn;t want to get too ahead of ourselves